ethers-solc/src/config.rs

```diff
@@ -538,7 +562,8 @@ impl ProjectPathsConfigBuilder {
                 .artifacts
                 .unwrap_or_else(|| ProjectPathsConfig::find_artifacts_dir(&root)),
             sources: self.sources.unwrap_or_else(|| ProjectPathsConfig::find_source_dir(&root)),
-            tests: self.tests.unwrap_or_else(|| root.join("tests")),
+            tests: self.tests.unwrap_or_else(|| root.join("test")),
```

Comment:
I changed here, because it seems like `tests` was a mistake here. Should I revert it?